### PR TITLE
Victorian/CourseraTest

### PR DESCRIPTION
**Pull request by vnvictoria**
Hw11

### CODE REVIEW

File: src/test/java/VictoriaTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    @Ignore
+    @Test
+    public void CourseraTest() {
+        getDriver().get("https://www.coursera.org/");
+        WebElement searchBox = getDriver().findElement(By.xpath("//div[@class='react-autosuggest__container']//input"));
+        searchBox.sendKeys("Test automation");
+        WebElement search = getDriver().findElement(By.xpath("//header/div[1]/div[1]/div[1]/div[1]/div[3]/div[1]/form[1]/div[1]/div[1]/div[1]/button[2]/div[1]/*[1]"));
+        search.click();
+        search = getDriver().findElement(By.xpath("//header/div[2]/div[1]/div[1]/div[1]/div[1]/div[3]/div[1]/div[2]/form[1]/div[1]/div[1]/div[1]/div[1]/input[1]"));
```
Comment (maratdp) [inline src/test/java/VictoriaTest.java:19]: Here as well. Try to change XPath even your one works

File: src/test/java/VictoriaTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class VictoriaTest extends BaseTest {
+
+    @Ignore
```
[the remark this answers is not in the document]
Comment (vnvictoria) [reply src/test/java/VictoriaTest.java:11]: Thanks, it is deleted now.

File: src/test/java/VictoriaTest.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    @Test
+    public void CourseraTest() {
+        getDriver().get("https://www.coursera.org/");
+        WebElement searchBox = getDriver().findElement(By.xpath("//div[@class='react-autosuggest__container']//input"));
+        searchBox.sendKeys("Test automation");
+        WebElement search = getDriver().findElement(By.xpath("//header/div[1]/div[1]/div[1]/div[1]/div[3]/div[1]/form[1]/div[1]/div[1]/div[1]/button[2]/div[1]/*[1]"));
```
[the remark this answers is not in the document]
Comment (vnvictoria) [reply src/test/java/VictoriaTest.java:17]: Thank you, I am still learning how to create shorter relative Xpaths.

File: src/test/java/VictoriaTest.java (hunk 1 above)
Comment (vnvictoria) [reply src/test/java/VictoriaTest.java:19]: I made changes as advised, please review. Thank you.

**Pull request closed without merging**